package.json

```diff
@@ -58,12 +59,68 @@
     "mapillary-js/src/Mapillary": "mapillary-js/dist/mapillary.js",
     "typeahead": "corejs-typeahead"
   },
+  "dependencies": {
+    "@fortawesome/fontawesome-free": "5.15.4",
+    "@lit/reactive-element": "2.0.4",
+    "@trevoreyre/autocomplete-js": "3.0.2",
+    "angular": "1.8.3",
+    "angular-animate": "1.8.3",
+    "angular-dynamic-locale": "0.1.38",
+    "angular-float-thead": "0.1.2",
+    "angular-gettext": "2.4.2",
+    "angular-gettext-tools": "2.5.3",
+    "angular-sanitize": "1.8.3",
+    "angular-touch": "1.8.3",
+    "angular-ui-date": "1.1.1",
+    "angular-ui-slider": "0.4.0",
+    "bootstrap": "4.6.2",
+    "core-js": "3.38.1",
+    "corejs-typeahead": "1.3.4",
+    "d3": "7.9.0",
+    "file-saver": "2.0.5",
+    "floatthead": "2.2.5",
+    "i18next": "23.14.0",
+    "i18next-browser-languagedetector": "8.0.0",
+    "i18next-http-backend": "2.6.1",
+    "i18next-parser": "9.0.2",
+    "jquery": "3.7.1",
+    "jquery-datetimepicker": "2.5.21",
+    "jquery-mousewheel": "3.1.13",
+    "jquery-ui": "1.14.0",
+    "jquery-ui-touch-punch": "0.2.3",
+    "lit": "3.2.0",
+    "lit-element": "4.1.0",
+    "lit-html": "3.2.0",
+    "loc-i18next": "0.1.6",
+    "localforage": "1.10.0",
+    "moment": "2.30.1",
+    "neat-csv": "6.0.1",
+    "ol": "10.1.0",
+    "ol-cesium": "2.14.0",
+    "ol-layerswitcher": "4.1.2",
+    "ol-mapbox-style": "12.3.5",
+    "proj4": "2.12.0",
+    "qruri": "0.0.4",
+    "regenerator-runtime": "0.14.1",
+    "resize-observer-polyfill": "1.5.1",
+    "rxjs": "7.8.1",
+    "tinycolor2": "1.6.0"
+  },
+  "peerDependencies": {
+    "cheerio": "1.0.0",
```

Comment:
What's for?

Reply:
It appears with this commit, without explication.
I think it was added to fix a sub-dependency, that's why I've added it as peerDependencies.

Reply:
Strange effectively, for me, we should try remove it

Reply:
Ok, I'll try to remove it